proposals/relaxed-simd/Overview.md

```diff
@@ -252,8 +252,6 @@ def q15mulr(a, b):
 
 - `i16x8.dot_i8x16_i7x16_s(a: v128, b: v128) -> v128`
 - `i16x8.dot_i8x16_i7x16_u(a: v128, b: v128) -> v128`
-- `i32x4.dot_i8x16_i7x16_add_s(a: v128, b:v128, c:v128) -> v128`
```

Comment:
The discussion in #52 removes  i16x8.dot_i8x16_i7x16_u and i32x4.dot_i8x16_i7x16_add_u -- we can keep i32x4.dot_i8x16_i7x16_add_s for now.

Reply:
Thanks, there is a typo in the description, the discussion there mentions removing unsigned dot products, updated.